### DISCUSSION TITLE
Unable to choose NIC management interface during setup

### DISCUSSION THREAD

**Discussion by varqasim** [Security-Onion-Solutions/securityonion#1915] [Unsupported Versions] [answered]
Installing SecurityOnion on Virtualbox on a MacOSX. When going through the initial setup I am unable to choose the Management NIC from the different networks that I have created through Virtualbox which are NAT, Bridged and Internet. Unlike the previous steps where one of the options that are presented is highlighted with an *, when it comes to the NIC Management page selection, none of the options are highlighted nor am I able to select one. If I do choose one and click OK, or just hit enter the page just loads again with the same selection.

<img width="384" alt="Screenshot 2020-11-13 at 19 09 13" src="https://user-images.githubusercontent.com/6570357/99093619-c8b45f00-25e3-11eb-8913-93a8cda6764c.png">

Any ideas what can lead to this issue?

**Comment by dougburks** [comment, marked as answer]
Have you tried using the space bar to select an item as described at https://docs.securityonion.net/en/2.3/configuration.html?

**Reply by varqasim** [reply]
Ah, that was it! Should've went through the docs more thoroughly.
